src/storage/kura.rs

```diff
@@ -1,4 +1,4 @@
-use crate::model;
+use crate::model::model;
```

Comment:
?

Reply:
Moved to model directory. This will likely be split up into more descriptive files later.